Sources/MapboxCoreNavigation/TileStoreLocation.swift

```diff
@@ -0,0 +1,64 @@
+
+import Foundation
+import MapboxCommon
+
+/**
+ To add docs for cases
+ */
+public enum TileStoreLocation {
+    public enum Strict {
+        case `default`
+        case custom(URL)
+        
+        public var tilStoreURL: URL? {
+            switch self {
+            case .default:
+                return nil
```

Comment:
Those enums seem too sophisticated IMHO, especially the fact that `Strict.tileStoreURL` can be nil.
I thought that `Strict` enum is used to explicitly show that the url is always present.
I wonder if there is a way to simplify code here.